rocAL/rocAL/include/meta_data/meta_data.h

```diff
@@ -36,25 +36,34 @@ THE SOFTWARE.
 #define SCALE_CONSTANT_HALF_BODY 1.5
 typedef struct BoundingBoxCord_
 {
-  float l; float t; float r; float b;
+  double l; double t; double r; double b;
   BoundingBoxCord_() {}
-  BoundingBoxCord_(float l_, float t_, float r_, float b_): l(l_), t(t_), r(r_), b(b_) {}   // constructor
+  BoundingBoxCord_(double l_, double t_, double r_, double b_): l(l_), t(t_), r(r_), b(b_) {}   // constructor
```

Comment:
check if we really need double for bbox coordinates.
Use only one structure for both (ltrb or xywh) and use a flag to differentiate

Reply:
Will check this and all support

Reply:
Ok. Please remove that from this PR since it is just confusing having two different struct names with same data inside